microshift_configuring/microshift-using-config-tools.adoc

```diff
@@ -0,0 +1,16 @@
+:_content-type: ASSEMBLY
+[id="microshift-using-config-tools"]
+= MicroShift Version
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-version
+toc::[]
+
+In {product-title}, you have the option of using the following tools, in order of precedence:
```

Comment:
Let's emphasize the configuration file and ignore the other options. The config file is more manageable (gitops, etc.) and we're likely going to remove at least the environment variables soon, so we can avoid having to make edits here when that happens.